### ISSUE TITLE
Manage multiple rows of query response

### ISSUE THREAD

**Issue by pbchase** [ctsit/data_driven_project_banner#16]
If the query returns multiple rows of data it would be nice to at least have the option to repeat the smart-variable replacement once for each row. e.g.,

```
These are your outstanding invoices:

Project_ID, Invoice_ID
2164, 1806252164
2164, 1907122164
```

In the current code, only the first row returned is used in the substitution.

Admittedly an alert like the one below would need to be re-written to convey the right message.

![old style invoice](https://user-images.githubusercontent.com/3356493/119394403-20710880-bca0-11eb-80ac-fd7fc7d34840.png)

**Comment by ChemiKyle** [comment]
Root of the issue is this function: https://github.com/ctsit/data_driven_project_banner/blob/cd7ed19972cc491b6d4803b289d89195ab814140/ExternalModule.php#L50-L67

It should operate over each item in `$sql_response` and return a `collapse`d string from the array output.

**Comment by ChemiKyle** [comment]
@pbchase how do you like this? Rich text field tables play quite nicely with the infrastructure we already have, this is not a mockup:

![Screen Shot 2021-06-03 at 4 02 53 PM](https://user-images.githubusercontent.com/20332546/120705202-840bea80-c485-11eb-82c8-1a0899288b3d.png)

If you like this I'll add some admin toggles for repeating data.

**Comment by ChemiKyle** [comment]
Addressed by #19
